### PR TITLE
minor updates/changes in txmempool.h/.cpp

### PR DESCRIPTION
**Pull request by Diapolo**

### CODE REVIEW

**Comment by paveljanik** [comment]
@Diapolo are you willing to maintain and constantly rebase the "cleanup" branch of bitcoin? Ie. include all const changes, whitespace changes, clang-format changes etc. in your own tree? Let's try it for a month or so...

File: src/txmempool.cpp [hunk 1]
```diff
@@ -750,7 +745,7 @@ void CTxMemPool::PrioritiseTransaction(const uint256 hash, const string strHash,
         deltas.first += dPriorityDelta;
         deltas.second += nFeeDelta;
     }
-    LogPrintf("PrioritiseTransaction: %s priority += %f, fee += %d\n", strHash, dPriorityDelta, FormatMoney(nFeeDelta));
+    LogPrintf("%s: %s priority += %f, fee += %d\n", __func__, strHash, dPriorityDelta, FormatMoney(nFeeDelta));
```
Comment (dcousens) [inline src/txmempool.cpp]: This hardly seems worth it

File: src/txmempool.cpp [hunk 2]
```diff
@@ -780,7 +775,8 @@ bool CTxMemPool::HasNoInputsOf(const CTransaction &tx) const
 
 CCoinsViewMemPool::CCoinsViewMemPool(CCoinsView *baseIn, CTxMemPool &mempoolIn) : CCoinsViewBacked(baseIn), mempool(mempoolIn) { }
 
-bool CCoinsViewMemPool::GetCoins(const uint256 &txid, CCoins &coins) const {
+bool CCoinsViewMemPool::GetCoins(const uint256 &txid, CCoins &coins) const
```
Comment (dcousens) [inline src/txmempool.cpp]: Your `&` syntax alternates too

**Comment by dcousens** [comment]
Trivial, utACK, but, meh

**Pull request closed without merging**